winit/src/application.rs

```diff
@@ -536,8 +543,10 @@ pub fn run_command<Message: 'static + std::fmt::Debug + Send, E: Executor>(
     clipboard: &mut Clipboard,
     proxy: &mut winit::event_loop::EventLoopProxy<Message>,
     window: &winit::window::Window,
+    graphics_info: &window::GraphicsInformation,
```

Comment:
I couldn't think of other ways to do this, I'd be happy to receive feedback on this.

Reply:
Done in https://github.com/iced-rs/iced/pull/1314/commits/6e167675d6a51a8a78d93439719ebffe35dcfdef.